.gitignore

```diff
@@ -478,3 +478,5 @@ $RECYCLE.BIN/
 .vscode/settings.json
 
 src/.idea/.idea.Nox.Generator/.idea/
+
+tests/Nox.Generator.Tests/Generated/
```

Comment:
Do we really need this?

Reply:
When I do implementation I generate files under Generated folder so that I can see what is generated exactly. 
Preventing commits by accident I amended gitignore.